mixin/Makefile

```diff
@@ -15,20 +17,62 @@ fmt:
 		xargs -n 1 -- $(JSONNET_FMT) -i
 
 .PHONY: lint
-lint: build
+lint: build lint-pint
+	@set -e; \
 	find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
 		while read f; do \
-			$(JSONNET_FMT) "$$f" | diff -u "$$f" -; \
-		done
+			$(JSONNET_FMT) "$$f" | diff -u "$$f" - || exit 1; \
+		done; \
 	mixtool lint mixin.libsonnet
 
-dashboards_out: mixin.libsonnet config.libsonnet $(wildcard dashboards/*)
+.PHONY: lint-pint
+lint-pint: prometheus_alerts.yaml prometheus_rules.yaml
+	pint lint --min-severity=info --fail-on=warning prometheus_rules_out/
+
+dashboards_out: mixin.libsonnet $(wildcard **/*.libsonnet) $(wildcard **/*.jsonnet) $(wildcard panels/*) $(wildcard signals/*)
 	@mkdir -p dashboards_out
 	mixtool generate dashboards mixin.libsonnet -d dashboards_out
 
-prometheus_alerts.yaml: mixin.libsonnet alerts/*.libsonnet
```

Comment:
This seems like a lot of changes to this file which I think if you just removed the `alerts/*.libsonnet` we shouldn't have to complicate this makefile too much